src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1079,7 +1079,10 @@ public virtual SignOutResult SignOut(AuthenticationProperties properties, params
                 throw new ArgumentNullException(nameof(prefix));
             }
 
-            return TryUpdateModelAsync(model, prefix, new CompositeValueProvider(ControllerContext.ValueProviders));
+            var valueProvider = await CompositeValueProvider.CreateAsync(
+                ControllerContext,
+                ControllerContext.ValueProviderFactories);
```

Comment:
I'm going to just remove the action context overload, since the whole point of this method is to reduce boilerplate. We can add it back if anyone has an important need.